README.md

```diff
@@ -1 +1,13 @@
-# ACCESS-OM3 payu configuration for 1 deg JRA55-do RYF
+# MOM6-CICE6-WW3 1 deg JRA55-do RYF ACCESS-OM3 configuration
+
+**WARNING: This configuration is still under development and should not be used for production.**
+
+## Features
+
+- data atmosphere (datm) = JRA55-do v1-4, RYF 1990-1991
+- data runoff (drof) = JRA55-do v1-4, RYF 1990-1991
```

Comment:
```suggestion
- data atmosphere (DATM) = JRA55-do v1-4, RYF 1990-1991
- data runoff (DATM) = JRA55-do v1-4, RYF 1990-1991
```

Reply:
Could also add
```
See [main branch README](https://github.com/COSIMA/MOM6-CICE6/blob/main/README.md) for usage information.
```
This will provide users with information they need without requiring us to provide it consistently in every branch.